Xamarin.ExposureNotification/CallbackService.android.cs

```diff
@@ -1,83 +1,38 @@
 ﻿using Android.App;
 using Android.Content;
-using Android.OS;
 using Android.Runtime;
 using AndroidX.Core.App;
 
 namespace Xamarin.ExposureNotifications
 {
-	// TODO: Verify if this is still needed
-	// Previously the spec suggested starting the API needed to be done each boot
-	// but it looks like in the latest revision this is not the case
-
-	//[Service(
-	//	DirectBootAware = true,
-	//	Permission = "android.permission.BIND_JOB_SERVICE")]
-	//internal class ExposureNotificationBootService : JobIntentService
-	//{
-	//	const int JobId = 0x01;
-
-	//	public static void EnqueueWork(Context context, Intent work)
-	//		=> JobIntentService.EnqueueWork(context, Java.Lang.Class.FromType(typeof(ExposureNotificationBootService)), JobId, work);
-
-	//	protected override async void OnHandleWork(Intent workIntent)
-	//		=> await ExposureNotification.StartAsync();
-	//}
-
-	//[BroadcastReceiver]
-	//[IntentFilter(new [] { Intent.ActionBootCompleted })]
-	//internal class ExposureNotificationBootBroadcastReceiver : BroadcastReceiver
-	//{
-	//	public override void OnReceive(Context context, Intent intent)
-	//	{
-	//		if (intent.Action == Intent.ActionBootCompleted)
-	//			ExposureNotificationBootService.EnqueueWork(context, new Intent());
-	//	}
-	//}
-
 	[BroadcastReceiver(Permission = "com.google.android.gms.nearby.exposurenotification.EXPOSURE_CALLBACK")]
-	[IntentFilter(new[]
-	{
-		ActionExposureStateUpdated,
-		ActionRequestDiagnosisKeys
-	})]
+	[IntentFilter(new[] { "com.google.android.gms.exposurenotification.ACTION_EXPOSURE_STATE_UPDATED" })]
 	[Preserve]
-	internal class ExposureNotificationCallbackBroadcastReceiver : BroadcastReceiver
+	class ExposureNotificationCallbackBroadcastReceiver : BroadcastReceiver
 	{
-		internal const string ActionExposureStateUpdated = "com.google.android.gms.exposurenotification.ACTION_EXPOSURE_STATE_UPDATED";
-		internal const string ActionRequestDiagnosisKeys = "com.google.android.gms.exposurenotification.ACTION_REQUEST_DIAGNOSIS_KEYS";
```

Comment:
Pre-emptively removing "ACTION_REQUEST_DIAGNOSIS_KEYS" as that is gone in v1.3 of the API